Sources_v3/StreamChatUI/ChatChannelList/ChatSwipeableListItemView.swift

```diff
@@ -0,0 +1,101 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class ChatSwipeableListItemView<ExtraData: ExtraDataTypes>: View, UIConfigProvider, UIGestureRecognizerDelegate {
+    // MARK: - Properties
+
+    private var startedValue: CGFloat = 0
+    public var trailingConstraint: NSLayoutConstraint?
+
+    public var deleteButtonAction: (() -> Void)?
+
+    /// Main Content view to which you should always embed your cell content.
+    public private(set) lazy var cellContentView: UIView = UIView().withoutAutoresizingMaskConstraints
```

Comment:
nit: Why "cell"? This is just a view

Reply:
I get your idea. I didn't want this to be confused with `contentView` of cell instance, because it could be dangerous...  But any suggestions are welcome :)